src/mono/wasm/runtime/binding_support.js

```diff
@@ -123,31 +123,29 @@ var BindingSupportLib = {
 
 			this._bind_js_obj = bind_runtime_method ("BindJSObject", "iii");
 			this._bind_core_clr_obj = bind_runtime_method ("BindCoreCLRObject", "ii");
-			this._bind_existing_obj = bind_runtime_method ("BindExistingObject", "mi");
-			this._unbind_raw_obj_and_free = bind_runtime_method ("UnBindRawJSObjectAndFree", "ii");
+			this._alloc_gc_handle = bind_runtime_method ("GetJSOwnedObjectHandle", "m");
```

Comment:
When defining new APIs let's try to keep the js and C# method names as close as possible, the existing code using wildly different names does no one any favors